src/api/endpoints/activity_types.py

```diff
@@ -0,0 +1,133 @@
+"""Module for activity types operations."""
+
+from flask import request
+from flask_restful import Resource
+
+from api.models import ActivityType
+from api.utils.auth import roles_required, token_required
+from api.utils.helpers import find_item, response_builder
+from api.utils.marshmallow_schemas import (activity_types_schema,
+                                           new_activity_type_schema)
+
+
+class ActivityTypesAPI(Resource):
+    """Activity Categories Resource."""
+
+    decorators = [token_required]
+
+    @classmethod
+    @roles_required(["success ops"])
+    def post(cls):
+        """Create new activity type."""
+        payload = request.get_json(silent=True)
+        if not payload:
+            return response_builder(dict(
+                                    message="Data for creation must "
+                                            "be provided.",
+                                    status="fail",
+                                    ), 400)
+
+        result, errors = new_activity_type_schema.load(payload)
+
+        if errors:
+            status_code = new_activity_type_schema.context.get(
+                            'status_code')
+            validation_status_code = status_code or 400
+            return response_builder(errors, validation_status_code)
+
+        support_mutiple = result["supports_multiple_participants"]
```

Comment:
v1.0.0.1 Correcting these typos

Reply:
@georgreen Needs that spell checker vs code extension in his life. You type too quickly.